src/Hit/Issue.hs

```diff
@@ -164,3 +168,18 @@ parseOwnerRepo url =
 
     stripGitSuffix :: Text -> Maybe Text
     stripGitSuffix x = whenNothing (T.stripSuffix ".git" x) (Just x)
+
+fetchIssueStrict :: Id Issue -> IO Issue
+fetchIssueStrict iNum = fetchIssue iNum >>= \case
+    Left err -> errorMessage (show err) >> exitFailure
+    Right issue -> pure issue
+
+assignToIssue :: Issue -> IO (Either Error Issue)
+assignToIssue issue = withOwnerRepo action
+  where action Nothing _ _ = pure $ Left $ UserError "GITHUB_TOKEN lookup failed"
+        action (Just t) o r = do
```

Comment:
According to our style guide, `where` should be on a separate line, functions inside `where` should  have 4 space indent and should have types. So the code should look like:

```haskell
  where
    action :: ...
    action = ...
```